src/config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * General configuration
+ *
+ * @class config
+ * @memberof dc
+ * @returns {dc.config}
+ */
+dc.config = (function () {
+    var _config = {};
+
+    var _useV2CompatColorScheme = false;
```

Comment:
I think we should default to true and issue the warning. This way people get backward compatibility by default, to ease their transition into the new world, but they are also informed that it's going away.

Reply:
Will do!